packages/syft/src/syft/service/worker/utils.py

```diff
@@ -283,15 +300,15 @@ def prepare_kubernetes_pool_env(runner: KubernetesRunner, env_vars: dict):
 
     # clone and patch backend environment variables
     backend_env = runner.get_pod_env_vars(backend_pod_name) or []
-    env_vars = KubeUtils.patch_env_vars(backend_env, env_vars)
+    env_vars_list = KubeUtils.patch_env_vars(backend_env, env_vars)
```

Comment:
A bit nitpick but when I encounter this I do this instead:

```py
env_vars_ = ... # or _env_vars
env_vars = ...
```

It's clearer that `env_vars_` and `env_vars` mean the same thing. Also I'm not a fan of having type in variable names haha.

Reply:
Thanks @kiendang. Also, is the underscore at the end of variable a convention in our codebase?

Reply:
I don't think there's a codebase wide convention more like my own convention, but I certainly did this in various places in the codebase. Anw this is not too important and highly opinionated so do as you see fit.

Reply:
This is probably the best option `env_vars_: List = ...`